.github/workflows/release.yaml

```diff
@@ -165,34 +158,34 @@ jobs:
 
-      - name: Build The Release Docker Images
-        run: ./.github/scripts/build-release-images-for-docker.sh ${{ github.event.inputs.branch }}
+      - name: Build The Images For Docker.io
+        run: cd registry && ./.github/scripts/build-images.sh ${{ github.event.inputs.branch }} docker.io release ${{ github.event.inputs.release-version}} 
 
-      - name: List All The Tagged Docker Images
+      - name: List All Docker Images
         run: docker images
           
-      - name: Push The Release Docker Images To Dockerhub 
-        run: ./.github/scripts/push-release-images-to-docker.sh ${{ github.event.inputs.branch }} ${{ github.event.inputs.release-version}}
+      - name: Push The Images To Docker.io 
+        run: cd registry && ./.github/scripts/push-images.sh ${{ github.event.inputs.branch }} docker.io release ${{ github.event.inputs.release-version}}
           
       - name: Login to Quay.io Registry
-        run: docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" quay.io
-      
-      - name: Tag Release Docker Images for Quay.io
-        run: ./.github/scripts/tag-release-images-for-quay.sh ${{ github.event.inputs.branch }} ${{ github.event.inputs.release-version}}
+        run: docker login -u "${{ secrets.QUAY_USERNAME }}" -p "${{ secrets.QUAY_PASSWORD }}" Quay.io
+        
+      - name: Build The Images For Quay.io
+        run: cd registry && ./.github/scripts/build-images.sh ${{ github.event.inputs.branch }} quay.io release ${{ github.event.inputs.release-version}} 
 
-      - name: List All The Tagged Docker Images
+      - name: List All Docker Images
         run: docker images
-
-      - name: Push The Release Docker Images to Quay.io
-        run: ./.github/scripts/push-release-images-to-quay.sh ${{ github.event.inputs.branch }} ${{ github.event.inputs.release-version}}
-
+          
+      - name: Push The Images To Quay.io 
+        run: cd registry && ./.github/scripts/push-images.sh ${{ github.event.inputs.branch }} quay.io release ${{ github.event.inputs.release-version}}
+ 
       - name: Maven Deploy
         run: |
           cd registry
           # Retry 3 times before the steps actually fails
-          (echo "===== Maven Deploy Attempt: 1 ====" && mvn deploy --batch-mode -Pmultitenancy -Pprod -Prelease -DskipTests --settings /home/runner/.m2/settings.xml -DskipTests) || \
-          (echo "===== Maven Deploy Attempt: 2 ====" && mvn deploy --batch-mode -Pmultitenancy -Pprod -Prelease -DskipTests --settings /home/runner/.m2/settings.xml -DskipTests) || \
-          (echo "===== Maven Deploy Attempt: 3 ====" && mvn deploy --batch-mode -Pmultitenancy -Pprod -Prelease -DskipTests --settings /home/runner/.m2/settings.xml -DskipTests) || \
+          (echo "===== Maven Deploy Attempt: 1 ====" && mvn deploy --batch-mode -Pmultitenancy -Pprod -Prelease -DskipTests --settings /home/runner/.m2/settings.xml) || \
```

Comment:
@EricWittmann I can see the storage profiles(-Psql & -Pkafkasql) are missing from the deploy command. is that intentional or?

Reply:
Yes that is intentional.  We don't want to deploy the actual applications to maven central.  Central is for re-usable libraries.  And the storages which are built in those profiles are the application variants.